keyboards/cannonkeys/tmov2/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "TMOv2",
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "width": 17,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":1.25, "y":0}, {"label":"Tab", "x":2.5, "y":0, "w":1.5}, {"label":"Q", "x":4, "y":0}, {"label":"W", "x":5, "y":0}, {"label":"E", "x":6, "y":0}, {"label":"R", "x":7, "y":0}, {"label":"T", "x":8, "y":0}, {"label":"Y", "x":9, "y":0}, {"label":"U", "x":10, "y":0}, {"label":"I", "x":11, "y":0}, {"label":"O", "x":12, "y":0}, {"label":"P", "x":13, "y":0}, {"label":"{", "x":14, "y":0}, {"label":"}", "x":15, "y":0}, {"label":"|", "x":16, "y":0}, {"x":1.25, "y":1}, {"label":"Caps Lock", "x":2.5, "y":1, "w":1.75}, {"label":"A", "x":4.25, "y":1}, {"label":"S", "x":5.25, "y":1}, {"label":"D", "x":6.25, "y":1}, {"label":"F", "x":7.25, "y":1}, {"label":"G", "x":8.25, "y":1}, {"label":"H", "x":9.25, "y":1}, {"label":"J", "x":10.25, "y":1}, {"label":"K", "x":11.25, "y":1}, {"label":"L", "x":12.25, "y":1}, {"label":":", "x":13.25, "y":1}, {"label":"\"", "x":14.25, "y":1}, {"label":"Enter", "x":15.25, "y":1, "w":1.75}, {"x":1.25, "y":2}, {"label":"Shift", "x":2.5, "y":2, "w":2.25}, {"label":"Z", "x":4.75, "y":2}, {"label":"X", "x":5.75, "y":2}, {"label":"C", "x":6.75, "y":2}, {"label":"V", "x":7.75, "y":2}, {"label":"B", "x":8.75, "y":2}, {"label":"N", "x":9.75, "y":2}, {"label":"M", "x":10.75, "y":2}, {"label":"<", "x":11.75, "y":2}, {"label":">", "x":12.75, "y":2}, {"label":"?", "x":13.75, "y":2}, {"label":"Shift", "x":14.75, "y":2, "w":1.25}, {"x":16, "y":2}, {"x":1.25, "y":3}, {"x":4.5, "y":3}, {"x":5.5, "y":3, "w":1.5}, {"x":7, "y":3, "w":2.25}, {"x":9.25, "y":3, "w":2.75}, {"x":12, "y":3, "w":1.5}, {"x":13.5, "y":3}]
```

Comment:
This gets rid of the dead space in the rendering.

![image](https://user-images.githubusercontent.com/18669334/67744724-88d20800-f9df-11e9-8037-cc5f25a472d6.png)

```suggestion
                        "layout": [{"x":0, "y":0}, {"label":"Tab", "x":1.25, "y":0, "w":1.5}, {"label":"Q", "x":2.75, "y":0}, {"label":"W", "x":3.75, "y":0}, {"label":"E", "x":4.75, "y":0}, {"label":"R", "x":5.75, "y":0}, {"label":"T", "x":6.75, "y":0}, {"label":"Y", "x":7.75, "y":0}, {"label":"U", "x":8.75, "y":0}, {"label":"I", "x":9.75, "y":0}, {"label":"O", "x":10.75, "y":0}, {"label":"P", "x":11.75, "y":0}, {"label":"{", "x":12.75, "y":0}, {"label":"}", "x":13.75, "y":0}, {"label":"Backspace", "x":14.75, "y":0}, {"x":0, "y":1}, {"label":"Caps Lock", "x":1.25, "y":1, "w":1.75}, {"label":"A", "x":3, "y":1}, {"label":"S", "x":4, "y":1}, {"label":"D", "x":5, "y":1}, {"label":"F", "x":6, "y":1}, {"label":"G", "x":7, "y":1}, {"label":"H", "x":8, "y":1}, {"label":"J", "x":9, "y":1}, {"label":"K", "x":10, "y":1}, {"label":"L", "x":11, "y":1}, {"label":":", "x":12, "y":1}, {"label":"\"", "x":13, "y":1}, {"label":"Enter", "x":14, "y":1, "w":1.75}, {"x":0, "y":2}, {"label":"Shift", "x":1.25, "y":2, "w":2.25}, {"label":"Z", "x":3.5, "y":2}, {"label":"X", "x":4.5, "y":2}, {"label":"C", "x":5.5, "y":2}, {"label":"V", "x":6.5, "y":2}, {"label":"B", "x":7.5, "y":2}, {"label":"N", "x":8.5, "y":2}, {"label":"M", "x":9.5, "y":2}, {"label":"<", "x":10.5, "y":2}, {"label":">", "x":11.5, "y":2}, {"label":"?", "x":12.5, "y":2}, {"label":"Shift", "x":13.5, "y":2, "w":1.25}, {"x":14.75, "y":2}, {"x":0, "y":3}, {"label":"Win", "x":3.25, "y":3}, {"label":"Alt", "x":4.25, "y":3, "w":1.5}, {"x":5.75, "y":3, "w":2.25}, {"x":8, "y":3, "w":2.75}, {"label":"Alt", "x":10.75, "y":3, "w":1.5}, {"label":"Menu", "x":12.25, "y":3}]
```